packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx

```diff
@@ -108,7 +108,7 @@ const DateFilterInputs: FC<FilterInputsProps<DateFilterRule>> = (props) => {
                         parseDate={(value) =>
                             moment(value, `YYYY-MM-DD, HH:mm:ss:SSS`).toDate()
                         }
-                        defaultValue={new Date()}
+                        defaultValue={new Date().toString()}
```

Comment:
is it better to use `toISOString` or a shorter format ? I don't even know why you need to do this

Reply:
blueprint changed the `defaultValue` typing from Date to string. I don't think it matter much how the string is formatted.